udev: explicitly sort udev properties

Although udev properties are sorted by libudev, according
to libudev/systemd upstream this behavior cannot be relied
upon. Sort them explicitly for consistent behavior.

Resolves: rhbz#1939970

See: https://github.com/redhat-performance/tuned/pull/333

--- a/tuned/hardware/device_matcher_udev.py
+++ a/tuned/hardware/device_matcher_udev.py
@@ -18,7 +18,7 @@ def match(self, regex, device):
 		except AttributeError:
 			items = device.items()
 
-		for key, val in list(items):
+		for key, val in sorted(list(items)):
 			properties += key + '=' + val + '\n'
 
 		return re.search(regex, properties, re.MULTILINE) is not None
